src/components/GridLayout.css

```diff
@@ -1,4 +1,3 @@
-.HeaderImage,
```

Comment:
Heb een apart component van de settings gemaakt. De settings zaten voorheen in het GridLayout component. Ik heb de settings dus verplaatst van Gridlayout naar eigen component. Veel css van gridlayout heb ik dus ook verplaatst naar SettingsMenu.css. Deze heb ik dus in de Gridlayout.css verwijderd